src/main/java/appeng/capabilities/Capabilities.java

```diff
@@ -45,6 +46,8 @@ private Capabilities()
 
 	public static Capability<IStorageMonitorableAccessor> STORAGE_MONITORABLE_ACCESSOR;
 
+	public static Capability<ISpatialDimension> SPATIAL_DIMENSION;
+
 	public static Capability<ITeslaConsumer> TESLA_CONSUMER;
 
 	public static Capability<ITeslaHolder> TESLA_HOLDER;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make TESLA_HOLDER a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)